Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -881,29 +883,81 @@ def template_test(template: str):
                         f' The available Templates for this instance: {", ".join(template_names)}.')
 
 
-@logger
+@polling_function(
+    name=demisto.command(),
+    interval=60,
+    timeout=1200
+)
+def pan_os_command_polling(args: dict, result):
```

Comment:
```suggestion
def pan_os_command_polling(args: dict, result: dict):
```

result is dict/list?